Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-210-adapters/2.12.2-SNAPSHOT/jspwiki-210-adapters-2.12.2-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-210-adapters/2.11.0/jspwiki-210-adapters-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.filters.BasicPageFilter  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC BasicPageFilter()
	===  UNCHANGED METHOD: PUBLIC void destroy(org.apache.wiki.WikiEngine)
	===  UNCHANGED METHOD: PUBLIC void destroy(org.apache.wiki.api.core.Engine)
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.WikiEngine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC void postSave(org.apache.wiki.WikiContext, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC void postSave(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String postTranslate(org.apache.wiki.WikiContext, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String postTranslate(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preSave(org.apache.wiki.WikiContext, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preSave(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preTranslate(org.apache.wiki.WikiContext, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preTranslate(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.plugin.WikiPlugin  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC java.lang.String execute(org.apache.wiki.api.core.Context, java.util.Map<java.lang.String,java.lang.String><java.lang.String,java.lang.String>)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.PluginException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String execute(org.apache.wiki.WikiContext, java.util.Map<java.lang.String,java.lang.String><java.lang.String,java.lang.String>)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.PluginException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.providers.WikiAttachmentAdapterProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiAttachmentAdapterProvider()
	===  UNCHANGED METHOD: PUBLIC void deleteAttachment(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC void deleteVersion(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.util.Collection<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> findAttachments(org.apache.wiki.api.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC java.io.InputStream getAttachmentData(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Attachment getAttachmentInfo(org.apache.wiki.api.core.Page, java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getProviderInfo()
	===  UNCHANGED METHOD: PUBLIC java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> getVersionHistory(org.apache.wiki.api.core.Attachment)
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> listAllChanged(java.util.Date)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> listAttachments(org.apache.wiki.api.core.Page)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC void moveAttachmentsForPage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC void putAttachmentData(org.apache.wiki.api.core.Attachment, java.io.InputStream)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.providers.WikiAttachmentProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.WikiProvider
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteAttachment(org.apache.wiki.attachment.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteVersion(org.apache.wiki.attachment.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.attachment.Attachment><org.apache.wiki.attachment.Attachment> findAttachments(org.apache.wiki.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.io.InputStream getAttachmentData(org.apache.wiki.attachment.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.attachment.Attachment getAttachmentInfo(org.apache.wiki.WikiPage, java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.attachment.Attachment><org.apache.wiki.attachment.Attachment> getVersionHistory(org.apache.wiki.attachment.Attachment)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.attachment.Attachment><org.apache.wiki.attachment.Attachment> listAllChanged(java.util.Date)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.attachment.Attachment><org.apache.wiki.attachment.Attachment> listAttachments(org.apache.wiki.WikiPage)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void moveAttachmentsForPage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void putAttachmentData(org.apache.wiki.attachment.Attachment, java.io.InputStream)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.providers.WikiPageAdapterProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiPageAdapterProvider()
	===  UNCHANGED METHOD: PUBLIC void deletePage(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC void deleteVersion(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.util.Collection<org.apache.wiki.api.search.SearchResult><org.apache.wiki.api.search.SearchResult> findPages(org.apache.wiki.api.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC java.util.Collection<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getAllChangedSince(java.util.Date)
	===  UNCHANGED METHOD: PUBLIC java.util.Collection<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getAllPages()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC int getPageCount()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Page getPageInfo(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getPageText(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC java.lang.String getProviderInfo()
	===  UNCHANGED METHOD: PUBLIC java.util.List<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getVersionHistory(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC void movePage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC boolean pageExists(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean pageExists(java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC void putPageText(org.apache.wiki.api.core.Page, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.providers.WikiPageProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.WikiProvider
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deletePage(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteVersion(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.search.SearchResult><org.apache.wiki.search.SearchResult> findPages(org.apache.wiki.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.WikiPage><org.apache.wiki.WikiPage> getAllChangedSince(java.util.Date)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.WikiPage><org.apache.wiki.WikiPage> getAllPages()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getPageCount()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.WikiPage getPageInfo(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getPageText(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.WikiPage><org.apache.wiki.WikiPage> getVersionHistory(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void movePage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean pageExists(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean pageExists(java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void putPageText(org.apache.wiki.WikiPage, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.WikiProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.WikiEngine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED ANNOTATION: java.lang.Deprecated
